modules/cnf-du-pod-mutation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-pod-mutation_{context}"]
+
+= Pod mutation
+
+The workload partitioning feature modifies pods that are annotated with `target.workload.openshift.io/{workload-type}` that are in a namespace that contains the `workload.openshift.io/allowed: {workload-type}` annotation.
+
+For example, a pod with the following CPU request specified would be mutated:
+----
+requests:
+  cpu:
+    400m
+----
+
+The requested CPU resource is replaced with a management cores resource:
+----
+requests:
+  management.workload.openshift.io/cores: 400
+----
+
+The value of `resources.workload.openshift.io/{container-name}` is calculated as:
+
+----
+shares == (request_in_millis * 1024) /1000
+----
+
+This results in a value of 409 in this example.
+
+An annotation is added with the same value:
```

Comment:
If you can indicate what does the mutation--adding the annotation--please provide it.  Then the sentence can be revised to use active voice.